### DISCUSSION TITLE
Magento .htaccess from repo gets overwritten after composer install

### DISCUSSION THREAD

**Discussion by Aquive** [deployphp/deployer#3199] [General]
Hi,

As the title suggest. The ./.htaccess and ./pub/.htaccess files are overwritten after composer install from the Magento base module. There is actually some discussion in several github issues to consider this a feature or a bug.

My question is how would I make sure I can "reset" it to the repository version after composer install? I tried below, but it does not work because it fails, saying it cannot find a git repository.

```
after('deploy:vendors', function () {
    cd('{{release_path}}');
    run('git checkout pub/.htaccess');
});

```

**Comment by peterjaap** [comment]
```
after('deploy:vendors', function () {
    upload('pub/.htaccess', '{{release_path}}/pub/.htaccess');
});
```
